### DISCUSSION TITLE
mod ideea for explorer

### DISCUSSION THREAD

**Discussion by sz3bbyla** [ramensoftware/windhawk-mods#1253] [Mod ideas]
![Screenshot 2024-11-16 184135](https://github.com/user-attachments/assets/d944b187-6ea7-4c48-8309-95be978e06c2)
I am not an programer and don't know to do staff but i add an picture and I think many people have this problem 
the 2 bars up are to big, and the left side marked icons have big space between .Many recomandations are to go to explorer options to enable Decrease space between items, but this picture is already taken aster apply this setting

**Comment by Anixx** [comment]
This is related to you settings, most people do not have such a big gap as you have.
This also can be fixed by Open Shell or various already existing Windhawk mods, such as Classic Explorer Treeview:

![изображение](https://github.com/user-attachments/assets/cb93b49f-356d-4ee9-89da-24d57199cff1)
